rust/auction/program/src/processor/create_auction.rs

```diff
@@ -44,6 +44,8 @@ pub struct CreateAuctionArgs {
     pub tick_size: Option<u64>,
     /// Add a minimum percentage increase each bid must meet.
     pub gap_tick_size_percentage: Option<u8>,
+    /// Auction name
+    pub name: [u8; 32],
```

Comment:
same thing as I told Vadim, you break older clients when you do this. You need to create a new create auction endpoint to support this new struct, and basically call the same code - otherwise older clients will get a borshio error when they hit this and they won't know why! Also, name should be an Option tag on the model, so that it can optionally not be set if using the older endpoint.